tools/run_tests/xds_k8s_test_driver/framework/test_app/client_app.py

```diff
@@ -291,7 +289,8 @@ def run(self,
             qps=25,
             metadata='',
             secure_mode=False,
-            print_response=False) -> XdsTestClient:
+            print_response=False,
+            config_mesh=None) -> XdsTestClient:
```

Comment:
nit: can we insert `config_mesh` between `secure_mode` and `print_response`? No real reason, just a nit - I was hoping `print_response` will always be last in the args, and inserted rows won't modify unrelated args.

Reply:
Done.